src/redux/reducers/authentication/index.js

```diff
@@ -12,6 +11,7 @@ import {
     AUTH_SUCCESS_WITH_FACEBOOK,
     AUTH_ERROR_WITH_FACEBOOK,
 } from '../../actions/authentication';
+import { SHOW_ERROR_MESSAGE } from '../../actions/app/show-error-message';
```

Comment:
not, there is already a reducer listen for this action why do you need set it in others reducers if you only show the error based on app reducer.